package.json

```diff
@@ -19,8 +19,8 @@
     "query-string": "6.13.1",
     "react": "16.13.1",
     "react-dom": "16.13.1",
-    "react-ga": "^3.1.2",
     "react-dropzone": "11.0.3",
+    "react-ga": "^3.1.2",
```

Comment:
Curious about your opinion on `^`. Over the years, when this was an angular app, we would have strange library bugs around unexpected minor version updates. I spent many days sorting through a grunt build bug that was introduced in a minor library update that was pulled in because we used `^` instead of the exact version we knew worked. For that reason, I've always pulled in the exact version.

Reply:
Huh, didn't notice the ```^```. It must have popped up when I re-ran ```npm install```. I don't think we need to have this, especially now that there's dependabot integration on Github